SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/StorageAdapter.cs

```diff
@@ -1,13 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Microsoft.AppCenter.Utils;
```

Comment:
I thought System* was first in sort order

Reply:
We use the default settings of Visual Studio now to be more friendly to external contributors.